.github/actions/diffs/action.yml

```diff
@@ -56,6 +59,8 @@ runs:
             - "examples/**"
             - "iota_programmability/**"
             - ".github/workflows/_move_tests.yml"
+          isMoveIDE:
+            - "external-crates/move/crates/move-analyzer/**"
```

Comment:
`move-analyzer` is dependent on several other Move crates:
```
move-command-line-common.workspace = true
move-compiler.workspace = true
move-ir-types.workspace = true
move-package.workspace = true
move-symbol-pool.workspace = true
```
Should we run this job in case if `external-crates` changed?

Reply:
I added the crates you mentioned, but can also use `external-crates` if you prefer that, didn't check what sub dependencies we have and how they could impact something

Reply:
I think we can use just `external-crates` because I believe the analyzer depends on many Move crates and we don't update them often. In this case, we will be sure that it works correctly.

Reply:
Changed